Noemi_first_try/index/style.css

```diff
@@ -0,0 +1,121 @@
+    body {
+      margin: 0;
+    }
+    header {
+      background-image: url("./mann_telefon.jpg");
+      background-size: cover;
+      height: 110px;
+      padding-top: 230px;
+      padding-left: 500px;
+      margin-bottom: 10px;
+    }
```

Comment:
Nice work with the background image! 👍 
I think it fits the header well and like the color contrast with the text. 

If you want to use different background images for the different HTML-pages, one way could be to use different id's or class names in the HTML-files, and in the CSS file to use different background-image url's but keep the rest of the styling the same as here.